grr-ui/src/js/main.js

```diff
@@ -1,3 +1,7 @@
+    function sayHello(){
```

Comment:
Is this relevant to be adding to our main code?

Reply:
nope, left it in by accident